### PR TITLE
[ja] translate OpenTelemetry primer

### PR DESCRIPTION
**Pull request by ymotongpoo**
translate the following files

* `content/ja/docs/concepts/_index.md`
* `content/ja/docs/concepts/observability-primer.md`

### CODE REVIEW

File: content/en/docs/languages/java/configuration.md
Comment (svrnm) [inline content/en/docs/languages/java/configuration.md:1]: this should not be part of this PR

Comment (ymotongpoo) [reply content/en/docs/languages/java/configuration.md:1]: rebased

**Comment by svrnm** [comment]
requested review from @open-telemetry/docs-ja-approvers. We still need to update the code owners file for that

File: content/ja/docs/concepts/observability-primer.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## オブザーバビリティとは何か {#what-is-observability}
+
+オブザーバビリティは、システムの内部構造を知らなくても、そのシステムについて質問することで、システムを外側から理解することを可能にします。
+さらに、真新しい問題、つまり「未知の未知」のトラブルシューティングや対処が容易になります。
+また、「なぜこのようなことが起こるのか」という疑問に答えるのにも役立ちます。
+
+システムに関してこれらの質問をするためには、アプリケーションが適切に計装されていなければなりません。
+つまり、アプリケーションのコードは、[トレース](/docs/concepts/signals/traces/)、[メトリクス](/docs/concepts/signals/metrics/)、[ログ](/docs/concepts/signals/logs/)などの[シグナル](/docs/concepts/signals/)を発しなければなりません。
+開発者が問題をトラブルシュートするために計装を追加する必要がないとき、アプリケーションは適切に計装されています。なぜなら開発者が必要な情報をすべて持っているということになるからです。
```
Comment (katzchang) [inline content/ja/docs/concepts/observability-primer.md:17]: この箇所はちょっと読みにくかった。

> 開発者が問題をトラブルシュートするために計装を追加する必要がないとき、アプリケーションは適切に計装されていると言えます。

のように「と言えます」を追加したほうが自然かもしれないです。

**Review by katzchang — COMMENTED** [review]
One comment was made on the Japanese wording. Other than that, no problems.

File: content/ja/docs/concepts/observability-primer.md (hunk 1 above)
Comment (ymotongpoo) [reply content/ja/docs/concepts/observability-primer.md:17]: 確かに。変更します！

**Comment by ymotongpoo** [comment]
@open-telemetry/docs-approvers This PR got approval from @open-telemetry/docs-ja-approvers so can anyone move this forward? Thanks!

**Comment by svrnm** [comment]
> @open-telemetry/docs-approvers This PR got approval from @open-telemetry/docs-ja-approvers so can anyone move this forward? Thanks!

Thanks for tagging @ymotongpoo, @katzchang needs to re-review and click on "approve" if no other changes are required

**Review by svrnm — APPROVED** [review]

**Review by katzchang — APPROVED** [review]

**Comment by svrnm** [comment]
Not sure why the text linter reports failure, those are files unrelated to this PR

**Comment by svrnm** [comment]
thank you @ymotongpoo and @katzchang

**Pull request merged**